build.gradle

```diff
@@ -137,7 +137,7 @@ mainClassName = 'org.opendatakit.briefcase.Launcher'
 jar {
     baseName = buildConfig.appName
     version = buildConfig.version
-    archiveName = baseName + ' ' + version + '.jar'
+    archiveName = 'ODK-Briefcase-' + version + '.jar'
```

Comment:
Was this change discussed somewhere? I imagine it will break some downstream tools.

Reply:
Oh man. @yanokwa just explained it to me. The releases on GitHub have dashes because spaces are replaced. This makes local builds match.

Ok, I approve. Weird.

Reply:
It was "discussed" here: https://github.com/opendatakit/briefcase/pull/634#discussion_r219316223